website/cue/reference/components/sources/syslog.cue

```diff
@@ -8,6 +8,8 @@ components: sources: syslog: {
 	classes: sources.socket.classes
 
 	features: {
+		acknowledgements: sources.socket.features.acknowledgements
```

Comment:
Seems like this should be hard-coded to `false`? I can't conceive how we'd accomplish a protocol-oblivious means of doing acknowledgements for `socket`-based sources.

Reply:
Sure. I basically just copied the pattern of the other features here.

Reply:
Yeah, understandable.  It's probably fine as-is and we'll hopefully get to the configuration schema to generate this stuff before it would ever matter, again, given how unlikely it is that we ever somehow were able to add generic ack support to the `socket` source. 😅